### PR TITLE
CLI edits

### PR DESCRIPTION
**Pull request by maiyahershey**
This PR addresses:
- Edits made to the CLI section of the documentation
- I edited the cli overview, and added another tabbed list for the available subcommands.
- I merged all subcommands under cli usage section.

### CODE REVIEW

File: docs/cli/overview.md [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
-The CLI beauty is that its functionalities are a terminal command distance away.
-By doing,
+The power of the Inductiva CLI lies in its simplicity —just a command away: 
 ```bash
 inductiva
 ```
+Now, you can choose from a list of available commands to run via the Inductiva (CLI) to manage your projects and interact with our API:
 
-you will obtain an overview of the available commands and their descriptions.
+````{eval-rst}
```
Comment (maiyahershey) [inline docs/cli/overview.md:14]: another of our test `sphinx-tabs` @IvanPombo

File: docs/cli/overview.md [hunk 2]
```diff
@@ -1,23 +1,40 @@
 # Overview
 
-Walking side by side with the Python client, the Inductiva CLI provides 
-a quick and simple interface to interact and perform the most common tasks from
-the terminal. The CLI allows users to quickly manage their tasks, resources and/or
-storage, while their simulations are still running remotely.
+The Inductiva Command Line Interface (CLI) offers a streamlined and efficient way
```
Comment (IvanPombo) [inline docs/cli/overview.md:3]: Love it!

File: docs/cli/overview.md [hunk 3]
```diff
@@ -1,23 +1,40 @@
...
-the terminal. The CLI allows users to quickly manage their tasks, resources and/or
-storage, while their simulations are still running remotely.
+The Inductiva Command Line Interface (CLI) offers a streamlined and efficient way 
+to interact with our platform directly from your terminal. Designed for ease of use, 
+the CLI enables you to manage simulations, resources, and data in real-time, even 
+while your simulations run in the cloud.
 
-The CLI beauty is that its functionalities are a terminal command distance away.
-By doing,
+The power of the Inductiva CLI lies in its simplicity —just a command away:
```
Comment (IvanPombo) [inline docs/cli/overview.md:8]:
```suggestion
The power of the Inductiva CLI lies in its simplicity just a command away: 
```

File: docs/cli/overview.md [hunk 4]
```diff
@@ -1,23 +1,40 @@
...
+The Inductiva Command Line Interface (CLI) offers a streamlined and efficient way 
+to interact with our platform directly from your terminal. Designed for ease of use, 
+the CLI enables you to manage simulations, resources, and data in real-time, even 
+while your simulations run in the cloud.
 
-The CLI beauty is that its functionalities are a terminal command distance away.
-By doing,
+The power of the Inductiva CLI lies in its simplicity —just a command away: 
 ```bash
 inductiva
```
Comment (IvanPombo) [inline docs/cli/overview.md:10]: Here, we should add probably a mention to the output of running this command.

**Review by IvanPombo — APPROVED** [review]
Small tweak to add the output of the command! Will leave it here the code to add soon. Else, LGTM!

File: docs/cli/overview.md (hunk 1 above)
Comment (sergiomsantos) [reply docs/cli/overview.md:14]: What is this doing? I don't see anything rendered on the final doc:

File: docs/cli/overview.md [hunk 5]
```diff
@@ -1,23 +1,40 @@
...
 ```
+Now, you can choose from a list of available commands to run via the Inductiva (CLI) to manage your projects and interact with our API:
```
Comment (sergiomsantos) [inline docs/cli/overview.md:12]: 80 char line width

File: docs/cli/overview.md [hunk 6]
```diff
@@ -1,23 +1,40 @@
...
+      Get an overview of your remote storage.
+
+````
+
+For detailed information on any command, you can always use the `--help` or `-h` flag 
+to further understand its usage:
+
+```bash
+inductiva --help
+```
```
Comment (sergiomsantos) [inline docs/cli/overview.md:40]: let's give a hint that this is a terminal command by listing the prompt:

````suggestion
```bash
$ inductiva --help
```
````

File: docs/cli/overview.md [hunk 7]
```diff
@@ -1,23 +1,40 @@
 # Overview
 
-Walking side by side with the Python client, the Inductiva CLI provides 
-a quick and simple interface to interact and perform the most common tasks from
-the terminal. The CLI allows users to quickly manage their tasks, resources and/or
-storage, while their simulations are still running remotely.
+The Inductiva Command Line Interface (CLI) offers a streamlined and efficient way 
+to interact with our platform directly from your terminal. Designed for ease of use, 
+the CLI enables you to manage simulations, resources, and data in real-time, even
```
Comment (sergiomsantos) [inline docs/cli/overview.md:5]: comma before final enumeration:

```suggestion
the CLI enables you to manage simulations, resources and data in real-time, even 
```

Also, not sure the final adverb is applicable 🤔

**Review by sergiomsantos — CHANGES_REQUESTED** [review]
The tab thing is not working

File: docs/cli/overview.md (hunk 1 above)
Comment (IvanPombo) [reply docs/cli/overview.md:14]: Maybe we need to add an extension, no @maiyahershey?

Comment (maiyahershey) [reply docs/cli/overview.md:14]: Yes exactly! I just don't know how to configure this one. We might need to add the extension to the config file so this renders properly. 
@IvanPombo @sergiomsantos

Comment (IvanPombo) [reply docs/cli/overview.md:14]: Add this: "sphinx_tabs.tabs" to the extensions in the conf.py file.

Comment (IvanPombo) [reply docs/cli/overview.md:14]: Screenshot 2024-02-21 at 11 13 47

Comment (maiyahershey) [reply docs/cli/overview.md:14]: otimoooo

File: docs/cli/overview.md (hunk 7 above)
Comment (maiyahershey) [reply docs/cli/overview.md:5]: @sergiomsantos you mean real-time?

File: docs/cli/overview.md (hunk 1 above)
Comment (IvanPombo) [reply docs/cli/overview.md:14]: This is the looks now, I think you can now try to manage the coloring based on the tutorials of sphinx-tabs.

**Review by sergiomsantos — APPROVED** [review]

**Pull request merged**